### PR TITLE
Convert to sublime color scheme

### PR DESCRIPTION
**Pull request by Honiix**
I would like to propose a new color scheme with :
- Use variables instead of RGB code, many colors that looked alike have been merged
- Global variables are set, including the ones introduced in last Sublime builds
- Hashed Syntax Highlighting for JSON keys
- Support natively any markups (Markdown and Restructured Text tested)
- New color palette with enhanced contrast
- Purple color for misspelled words
- ... many other small changes

The original color scheme is kept in glacier-legacy.tmTheme

### CODE REVIEW

**Comment by joeyfigaro** [comment]
Thank you for doing this, @Honiix! I'll take a look at this over lunch.

File: code-samples/sample.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "deep 1": "test",
+    "Sequence": ["item1", "item2"],
+    "SequenceWithValues": 
+    [
+        {
+            "key": "value",
+            "key2": "value2",
+        }
+    ],
+    "glosrsay": {
```
Comment (joeyfigaro) [inline code-samples/sample.json]: Is this guy a typo?

Comment (Honiix) [reply code-samples/sample.json]: Yep, it's a typo, could be "Glossary". But it's just a sample file used to test the colors. It has no impact on the theme.

Comment (joeyfigaro) [reply code-samples/sample.json]: Cool, thanks for clarifying :100:

**Pull request merged**